sda/local.go

```diff
@@ -398,6 +398,11 @@ func (l *LocalTest) NewLocalConode(port int) *Conode {
 
 }
 
+// NewContext creates a context from a ServerIdentity.
+func (l *LocalTest) NewContext(si *network.ServerIdentity, servID ServiceID, manager *serviceManager) *Context {
```

Comment:
Hmm, looks like one needs to add a yet another public method anyways. But I agree that `NewContext` isn't the best name, here (sounds more like a 'stand-alone constructor' than a method on `LocalTest`). `Context` (preferred) or `GetContext` are better.